### DISCUSSION TITLE
Apparently, the scam bots have made their way to GitHub.

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#998] [General]
I was checking my GH notifications where I saw I had a notification from a discussion I started a week ago in a project of mine.

This is what it looked it looked like (PLEASE don't visit the links)

![image](https://user-images.githubusercontent.com/76978184/209886283-3ca5e2af-dbe9-432c-aba7-5a5aae81eb39.png)

My god why...

Using this I've reported, blocked and deleted the comment and it's commentor from my discussion post but goddamn now we need a GitHub-Spammer-Purge?

![image](https://user-images.githubusercontent.com/76978184/209886561-ac36e1d3-6cbd-440b-8162-1e8897bc4733.png)

**Comment by TechStudent10** [comment]
The account this was sent from was created this month, and so are most, if not, all the other accounts pinged.

**Reply by Firecul** [reply]
Not just this repo, looks like others have got a wave also https://github.com/microsoft/vscode-python/ (I've never contributed to this vscode-python repo so they seem to be targeting random repos and people)

**Reply by TechStudent10** [reply]
Yeah. Hopefully GH doesn't pull a YouTube and does nothing about it.

**Reply by bmlzootown** [reply]
>  (I've never contributed to this repo so they seem to be targeting random repos and people)

Same here, never contributed to this repo either. Got the ping while at work. Real question is, what's the point of this type of spam here of all places? I would hope it'd be a complete waste of their time.

**Reply by Firecul** [reply]
Sorry, I didn't write clearly. I didn't contribute to the vscode-python one.

I mean I guess if they do tag proper accounts it'll bring visibility here, **that** part isn't bad.

**Reply by TechStudent10** [reply]
> Sorry, I didn't write clearly. I didn't contribute to the vscode-python one.
> 
> I mean I guess if they do tag proper accounts it'll bring visibility here, **that** part isn't bad.

A GitHub staff member responded to me here: https://github.com/community/community/discussions/42869

**Comment by subnerf** [comment]
um wut is happing

**Reply by Firecul** [reply]
>  You are receiving this because you were mentioned.Message ID: ***@***.*** com>

I'm guessing you were one of the people tagged by the spam bot earlier so until you unsubscribe it'll keep letting you know of comments here.
